master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/policy-only/1.6/calico.yaml

```diff
@@ -184,6 +184,87 @@ spec:
           hostPath:
             path: /etc/cni/net.d
 
+# Create all the CustomResourceDefinitions needed for Calico
+---
+
+apiVersion: apiextensions.k8s.io/v1beta1
+description: Calico Felix Global Felix Configuration
+kind: CustomResourceDefinition
+metadata:
+   name: globalfelixconfigs.crd.projectcalico.org
+spec:
+  scope: Cluster
+  group: crd.projectcalico.org
+  version: v1
+  names:
+    kind: GlobalFelixConfig
+    plural: globalfelixconfigs
+    singular: globalfelixconfig
+
+---
+
+apiVersion: apiextensions.k8s.io/v1beta1
+description: Calico BGP Peers
+kind: CustomResourceDefinition
+metadata:
+  name: bgppeers.crd.projectcalico.org
```

Comment:
Are bgppeers, globalbgpconfigs, and ippools needed even though this manifest is for policy only?